🐛 Bug Report: SonarQube plugin does not compile

### 📜 Description

Updating to v1.11.0 breaks compilation in plugin-sonarqube-react

### 👍 Expected behavior

tsc compiles correctly

### 👎 Actual Behavior with Screenshots

yarn run v1.22.19
$ tsc --skipLibCheck false --incremental false
node_modules/@backstage/plugin-sonarqube/dist/index.d.ts:13:10 - error TS2305: Module '"@backstage/plugin-sonarqube-react"' has no exported member 'FindingSummary'.

13 import { FindingSummary } from '@backstage/plugin-sonarqube-react';
            ~~~~~~~~~~~~~~

node_modules/@backstage/plugin-sonarqube/dist/index.d.ts:16:10 - error TS2305: Module '"@backstage/plugin-sonarqube-react"' has no exported member 'SonarQubeApi'.

16 import { SonarQubeApi } from '@backstage/plugin-sonarqube-react';
            ~~~~~~~~~~~~

Found 2 errors in the same file, starting at: node_modules/@backstage/plugin-sonarqube/dist/index.d.ts:13

error Command failed with exit code 1.


### 👟 Reproduction steps

Update to backstage v1.11.0
Run `tsc --skipLibCheck false --incremental false`

### 📃 Provide the context for the Bug.

_No response_

### 🖥️ Your Environment

OS:   Darwin 22.1.0 - darwin/arm64
node: v16.19.0
yarn: 1.22.19
cli:  0.22.2 (installed)
backstage:  1.11.0

Dependencies:
  @backstage/app-defaults                                   1.2.0
  @backstage/backend-app-api                                0.4.0
  @backstage/backend-common                                 0.17.0, 0.18.2
  @backstage/backend-dev-utils                              0.1.0
  @backstage/backend-plugin-api                             0.4.0
  @backstage/backend-tasks                                  0.4.3
  @backstage/backend-test-utils                             0.1.34
  @backstage/catalog-client                                 1.3.1
  @backstage/catalog-model                                  1.2.0
  @backstage/cli-common                                     0.1.11
  @backstage/cli                                            0.22.2
  @backstage/config-loader                                  1.1.8
  @backstage/config                                         1.0.6
  @backstage/core-app-api                                   1.5.0
  @backstage/core-components                                0.11.2, 0.12.4
  @backstage/core-plugin-api                                1.4.0
  @backstage/dev-utils                                      1.0.12
  @backstage/errors                                         1.1.4
  @backstage/eslint-plugin                                  0.1.0
  @backstage/integration-aws-node                           0.1.1
  @backstage/integration-react                              1.1.10
  @backstage/integration                                    1.4.2
  @backstage/plugin-api-docs                                0.9.0
  @backstage/plugin-app-backend                             0.3.42
  @backstage/plugin-auth-backend                            0.18.0
  @backstage/plugin-auth-node                               0.2.11
  @backstage/plugin-catalog-backend-module-bitbucket-server 0.1.7
  @backstage/plugin-catalog-backend-module-msgraph          0.5.0
  @backstage/plugin-catalog-backend                         1.7.2
  @backstage/plugin-catalog-common                          1.0.11
  @backstage/plugin-catalog-graph                           0.2.27
  @backstage/plugin-catalog-import                          0.9.5
  @backstage/plugin-catalog-node                            1.3.3
  @backstage/plugin-catalog-react                           1.3.0
  @backstage/plugin-catalog                                 1.8.0
  @backstage/plugin-github-actions                          0.5.15
  @backstage/plugin-jenkins-backend                         0.1.32
  @backstage/plugin-jenkins-common                          0.1.13
  @backstage/plugin-jenkins                                 0.7.14
  @backstage/plugin-org                                     0.6.5
  @backstage/plugin-permission-common                       0.7.3
  @backstage/plugin-permission-node                         0.7.5
  @backstage/plugin-permission-react                        0.4.10
  @backstage/plugin-proxy-backend                           0.2.36
  @backstage/plugin-scaffolder-backend                      1.11.0
  @backstage/plugin-scaffolder-common                       1.2.5
  @backstage/plugin-scaffolder-node                         0.1.0
  @backstage/plugin-scaffolder-react                        1.1.0
  @backstage/plugin-scaffolder                              1.11.0
  @backstage/plugin-search-backend-module-elasticsearch     1.1.3
  @backstage/plugin-search-backend-module-pg                0.5.3
  @backstage/plugin-search-backend-node                     1.1.3
  @backstage/plugin-search-backend                          1.2.3
  @backstage/plugin-search-common                           1.2.1
  @backstage/plugin-search-react                            1.5.0
  @backstage/plugin-search                                  1.1.0
  @backstage/plugin-sonarqube-react                         0.1.2
  @backstage/plugin-sonarqube                               0.6.3
  @backstage/plugin-techdocs-backend                        1.5.3
  @backstage/plugin-techdocs-module-addons-contrib          1.0.10
  @backstage/plugin-techdocs-node                           1.5.0
  @backstage/plugin-techdocs-react                          1.1.3
  @backstage/plugin-techdocs                                1.5.0
  @backstage/plugin-user-settings                           0.7.0
  @backstage/release-manifests                              0.0.8
  @backstage/test-utils                                     1.2.5
  @backstage/theme                                          0.2.17
  @backstage/types                                          1.0.2
  @backstage/version-bridge                                 1.0.3


### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

None

--- a/.changeset/swift-bags-wave.md
+++ b/.changeset/swift-bags-wave.md
@@ -1,0 +1,6 @@
+---
+'@backstage/plugin-sonarqube-react': patch
+'@backstage/plugin-sonarqube': patch
+---
+
+Moved imports from `/alpha` to main public exports.

--- a/plugins/sonarqube-react/api-report.md
+++ b/plugins/sonarqube-react/api-report.md
@@ -6,7 +6,7 @@
 import { ApiRef } from '@backstage/core-plugin-api';
 import { Entity } from '@backstage/catalog-model';
 
-// @alpha (undocumented)
+// @public (undocumented)
 export interface FindingSummary {
   // (undocumented)
   getComponentMeasuresUrl: SonarUrlProcessorFunc;
@@ -25,7 +25,7 @@ export interface FindingSummary {
 // @public (undocumented)
 export const isSonarQubeAvailable: (entity: Entity) => boolean;
 
-// @alpha (undocumented)
+// @public (undocumented)
 export type MetricKey =
   | 'alert_status'
   | 'bugs'
@@ -39,7 +39,7 @@ export type MetricKey =
   | 'coverage'
   | 'duplicated_lines_density';
 
-// @alpha
+// @public
 export type Metrics = {
   [key in MetricKey]: string | undefined;
 };
@@ -47,7 +47,7 @@ export type Metrics = {
 // @public (undocumented)
 export const SONARQUBE_PROJECT_KEY_ANNOTATION = 'sonarqube.org/project-key';
 
-// @alpha (undocumented)
+// @public (undocumented)
 export type SonarQubeApi = {
   getFindingSummary(options: {
     componentKey?: string;
@@ -55,13 +55,13 @@ export type SonarQubeApi = {
   }): Promise<FindingSummary | undefined>;
 };
 
-// @alpha (undocumented)
+// @public (undocumented)
 export const sonarQubeApiRef: ApiRef<SonarQubeApi>;
 
-// @alpha (undocumented)
+// @public (undocumented)
 export type SonarUrlProcessorFunc = (identifier: string) => string;
 
-// @alpha
+// @public
 export const useProjectInfo: (entity: Entity) => {
   projectInstance: string | undefined;
   projectKey: string | undefined;

--- a/plugins/sonarqube-react/src/api/SonarQubeApi.ts
+++ b/plugins/sonarqube-react/src/api/SonarQubeApi.ts
@@ -16,7 +16,7 @@
 
 import { createApiRef } from '@backstage/core-plugin-api';
 
-/** @alpha */
+/** @public */
 export type MetricKey =
   // alert status
   | 'alert_status'
@@ -43,11 +43,11 @@ export type MetricKey =
   // duplicated lines
   | 'duplicated_lines_density';
 
-/** @alpha */
+/** @public */
 export type SonarUrlProcessorFunc = (identifier: string) => string;
 
 /**
- * @alpha
+ * @public
  *
  * Define a type to make sure that all metrics are used
  */
@@ -55,7 +55,7 @@ export type Metrics = {
   [key in MetricKey]: string | undefined;
 };
 
-/** @alpha */
+/** @public */
 export interface FindingSummary {
   lastAnalysis: string;
   metrics: Metrics;
@@ -65,12 +65,12 @@ export interface FindingSummary {
   getSecurityHotspotsUrl: () => string;
 }
 
-/** @alpha */
+/** @public */
 export const sonarQubeApiRef = createApiRef<SonarQubeApi>({
   id: 'plugin.sonarqube.service',
 });
 
-/** @alpha */
+/** @public */
 export type SonarQubeApi = {
   getFindingSummary(options: {
     componentKey?: string;

--- a/plugins/sonarqube-react/src/hooks/useProjectInfo.ts
+++ b/plugins/sonarqube-react/src/hooks/useProjectInfo.ts
@@ -25,7 +25,7 @@ export const SONARQUBE_PROJECT_INSTANCE_SEPARATOR = '/';
  *
  * If part or all info are not found, they will default to undefined
  *
- * @alpha
+ * @public
  * @param entity - entity to find the sonarqube information from.
  * @returns a ProjectInfo properly populated.
  */

--- a/plugins/sonarqube/api-report.md
+++ b/plugins/sonarqube/api-report.md
@@ -42,7 +42,7 @@ export const SonarQubeCard: (props: {
   duplicationRatings?: DuplicationRating[];
 }) => JSX.Element;
 
-// @alpha (undocumented)
+// @public (undocumented)
 export class SonarQubeClient implements SonarQubeApi {
   constructor({
     discoveryApi,

--- a/plugins/sonarqube/src/api/SonarQubeClient.ts
+++ b/plugins/sonarqube/src/api/SonarQubeClient.ts
@@ -23,7 +23,7 @@ import {
 import { InstanceUrlWrapper, FindingsWrapper } from './types';
 import { DiscoveryApi, IdentityApi } from '@backstage/core-plugin-api';
 
-/** @alpha */
+/** @public */
 export class SonarQubeClient implements SonarQubeApi {
   discoveryApi: DiscoveryApi;
   identityApi: IdentityApi;

